src/basic/log.c

```diff
@@ -1278,23 +1278,19 @@ int log_set_max_level_from_string(const char *e) {
         return 0;
 }
 
-int log_max_levels_to_string(int level, char **ret) {
+int log_max_levels_to_string(int clamp_level, char **ret) {
```

Comment:
This isn't only about clamping, but can actually be used to bump the log level of sd-executor. Also in `exec_invoke` we calls `log_set_max_level(context->log_level_max)`, so this makes things asymmetric...